lib/shared/messages.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
 
     flashComplete: _.template([
       '<%= imageBasename %> was successfully written to',
-      '<%= drive.description %> (<%= drive.name %>)'
+      '<%= drive.description %> (<%= drive.device %>)'
```

Comment:
We use to manually add the `.name` property in the drive scanner, which contained the device path in Linux and macOS, and the list of mounted volumes joined by commas on Windows. In order to not duplicate that logic on the CLI, I moved the addition of the name property to `drivelist`, and upgraded it here.